src/schema/date_time_options.rs

```diff
@@ -0,0 +1,287 @@
+use std::ops::BitOr;
+
+use serde::{Deserialize, Serialize};
+
+use super::Cardinality;
+use crate::schema::flags::{FastFlag, IndexedFlag, SchemaFlagList, StoredFlag};
+
+/// DateTime Precision
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+pub enum DatePrecision {
```

Comment:
`"precision": "Milliseconds" `
does not look great in a JSON.

`"precision": "milliseconds" would look better IMHO

can you add attriubte to make the serde serialization look nicer?
(serde lower should do the job.)